### PR TITLE
ERC20 transfer to ETH address documentation update/clarification added

### PR DESCRIPTION
**Pull request by blackwidoq**
some parts clarified and updated.

### CODE REVIEW

**Comment by blackwidoq** [comment]
@yash-sharma1   Hello, I made changes to the previous tutorial (`avax-c-chain-to-eth-address-transfer-tutorial.md`), adding example outputs, instructions on creating `.js` file and how to run the script in the terminal, etc.

File: network-documentation/avalanche/tutorials/erc20-token-transfer-to-eth-address.md [hunk 1]
```diff
@@ -213,6 +213,84 @@ async function PNG_getBalance() {
 
 Remember that `ether` in `convrted_balance` above is just to massage the balance in a more presentable form. \(1 ether = 10^9 Gwei\)
 
+At this point, you have gone through the entire script. 
+
+To run the script `ERC20_fromC_to_ETH_address.js`, type `node ERC20_fromC_to_ETH_address.js` and run it in your terminal (`node` before the file name is for NodeJs runtime.environment).
+
+The finished script should look as follows:
```
Comment (yash-sharma1) [inline network-documentation/avalanche/tutorials/erc20-token-transfer-to-eth-address.md:220]: The entire script file should be added above before the execution step so shift it above and after that the final execution step in both the tutorials

File: network-documentation/avalanche/tutorials/avax-c-chain-to-eth-address-transfer-tutorial.md [hunk 2]
```diff
@@ -102,6 +113,55 @@ main().catch((err) => {
...
 })
 ```
+At this point, the transfer should have completed. `console.log(`Transaction successful with hash: ${createReceipt.transactionHash}`)` outputs the transaction has in your terminal. This is your transaction record. An example output is shown below.
+
+![example](https://i.imgur.com/6uoCE6w.png)  
+
+To run the script `AVAX_C_chain_to_ETH_address.js`, type `node AVAX_C_chain_to_ETH_address.js` and run it in your terminal (`node` before the file name is for NodeJs runtime.environment).
+
+The finished script should look as follows:
+
```
Comment (yash-sharma1) [inline network-documentation/avalanche/tutorials/avax-c-chain-to-eth-address-transfer-tutorial.md:123]: The entire script file should be added above before the execution step so shift it above and after that the final execution step in both the tutorials

**Review by yash-sharma1 — CHANGES_REQUESTED** [review]

**Comment by blackwidoq** [comment]
@guillaumegaluz  Ok. I thought of asking people to go on [Pangolin](https://pangolin.exchange/) and swap AVAX for PNG or some other ERC20 tokens. But, I think it's kind of scary when a tutorial involves real money. So, I thought of asking people to go on Pangolin on Fuji, instead of Mainnet, to get some ERC20 tokens. However, when I looked up the [PNG address](https://github.com/pangolin-community/tokenlists/blob/main/fuji.tokenlist.json) on Fuji, I realized the Pangolin dex pretty much ran out of the test PNG tokens. So, I think I am going to follow [the official AVA lab ERC20 mint tutorial](https://docs.avax.network/build/tutorials/smart-digital-assets/create-erc-20-token-on-avalanche-c-chain), mint a large quantity of ERC-20 token and make them available on Pangolin by creating a liquidity pool and walk people through 1. how to go on Pangolin dex, 2. How to find the token I minted and swap FUJI AVAX for the example ERC-20 tokens 3. How to transfer the newly acquired tokens from one's Metamask to the C-chain of one's AVAX wallet 4. How to have it show up on your AVAX wallet.  

I guess what I outlined above can be the starting point, as opposed to assuming that one already has ERC-20 tokens in one's wallet.   I might actually create a Faucet instead of having people go on Pangolin and swap Fuji AVAX for the ERC-20 tokens.

**Comment by yash-sharma1** [comment]
> @guillaumegaluz Ok. I thought of asking people to go on [Pangolin](https://pangolin.exchange/) and swap AVAX for PNG or some other ERC20 tokens. But, I think it's kind of scary when a tutorial involves real money. So, I thought of asking people to go on Pangolin on Fuji, instead of Mainnet, to get some ERC20 tokens. However, when I looked up the [PNG address](https://github.com/pangolin-community/tokenlists/blob/main/fuji.tokenlist.json) on Fuji, I realized the Pangolin dex pretty much ran out of the test PNG tokens. So, I think I am going to follow [the official AVA lab ERC20 mint tutorial](https://docs.avax.network/build/tutorials/smart-digital-assets/create-erc-20-token-on-avalanche-c-chain), mint a large quantity of ERC-20 token and make them available on Pangolin by creating a liquidity pool and walk people through 1. how to go on Pangolin dex, 2. How to find the token I minted and swap FUJI AVAX for the example ERC-20 tokens 3. How to transfer the newly acquired tokens from one's Metamask to the C-chain of one's AVAX wallet 4. How to have it show up on your AVAX wallet.
> 
> I guess what I outlined above can be the starting point, as opposed to assuming that one already has ERC-20 tokens in one's wallet. I might actually create a Faucet instead of having people go on Pangolin and swap Fuji AVAX for the ERC-20 tokens.

Hey @blackwidoq 
Actually, we were working on few things to make tutorials more consistent and well structured. So moving further I request you please follow this guide **[HERE](https://learn.figment.io/other/tutorial-guidelines)** to make some required changes in your PR.  Also, please add the extra info and explanation in the tutorial as per the requested changes. Let us know if you have any queries.

**Comment by blackwidoq** [comment]
@yash-sharma1 @guillaumegaluz  Apparently, after one of the recent updates done by Avalanche, the old script for ERC-20 token transfer resulted in an error.  The same error was reported by multiple people. I think it has to do with the Berlin update being applied to the C-chain? Anyhow, I had to find a new way to do it. So, the tutorial is quite different from the old one but it achieves the same objective. Plus, now we do everything within one library (ethers), which is nice.  I outlined how to get a dummy ERC-20 token into one's Avalanche wallet C-chain as well.

**Review by yash-sharma1 — CHANGES_REQUESTED** [review]
Thanks for the quick fix @blackwidoq 

few points on this 
- About the Author section should be at the end of the tutorial. I request you to please give a thorough review to **[Sample Tutorial Structure](https://learn.figment.io/other/tutorial-guidelines/sample-tutorial-structure)** it'll help you in understanding the structure. 
- The introduction should be in 5-6 lines and it should contain what you're building/doing, what's the goal of the tutorial and for what purpose you're using these mentioned libraries etc.
- Make sure to take the error-driven approach
- At the end final script should be mentioned before showing the final output.
- Please follow the Sample Tutorial Structure and guidelines to make it consistent.

**Comment by pyr0gan** [comment]
> Thanks for the quick fix @blackwidoq

@blackwidoq any movement on this? 😄 Just want to keep things from getting stale.

**Comment by pyr0gan** [comment]
@blackwidoq We might need to close this PR. Can you confirm if you plan on completing it, please? 😄

**Comment by blackwidoq** [comment]
@pyr0gan Hi, sorry for the delay. I do intend to finish it. I am actually writing my phd thesis at the moment and have my defense late August. would it be ok if I finish the work after my defense?

**Comment by pyr0gan** [comment]
> @pyr0gan Hi, sorry for the delay. I do intend to finish it. I am actually writing my phd thesis at the moment and have my defense late August. would it be ok if I finish the work after my defense?

It will be here waiting when you get back. Best of luck defending your PhD thesis, that's some heavy work! 😅

**Pull request closed without merging**